### DISCUSSION TITLE
email/password with TOTP example for next.js

### DISCUSSION THREAD

**Discussion by robandrose** [nhost/nhost#1018] [Q&A]
Is there an example that can be used as a base to develop a totp login process?

**Comment by plmercereau** [comment]
Hello Matthias,
You can have a look at how it works with React in the [live react-apollo example](https://react-apollo.example.nhost.io/sign-in).
Here's the related [MFA TOTP code](https://github.com/nhost/nhost/blob/main/examples/react-apollo/src/profile/mfa.tsx). I don't think the logic will change between NextJs and React.
Good luck!

**Reply by osseonews** [reply]
The react example is great. Just wish it was done in Tailwind.

**Reply by plmercereau** [reply]
A small react tailwind example would be a nice addition indeed! Contributions are most welcome 🤗
